hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -234,15 +250,23 @@ set_false_path -through [get_pins top_earlgrey/u_spi_device/cio_csb_i] \
 # During pre-layout, the SPI_HOST_CLK source latencies are estimanted to account for
 # pad and logic latencies.  After CTS, source latency must be removed as all clocks are propagated
 
-set CLK_DIV_PIN top_earlgrey/u_spi_host0/u_spi_core/u_fsm/u_sck_flop/gen_*u_impl*/gen_flops[0]*.u_size_only_reg/Q
+# This requires knowledge of actual pin names, which are different depending on
+# whether we run this with tech libs or not.
+if {$FOUNDRY_ROOT != ""} {
+    set REG_PIN  gen_flops[0]*.u_size_only_reg/Q
+} else {
+    set REG_PIN  q_o[0]
+}
+
+set CLK_DIV_PIN top_earlgrey/u_spi_host0/u_spi_core/u_fsm/u_sck_flop/gen_*u_impl*/${REG_PIN}
 
 # internal divided by 2 clock (fastest spi host configuration)
 create_generated_clock -name SPI_HOST_INT_CLK -source [get_pins ${IO_CLK_PIN}] \
-    -divide_by 2 [get_pins ${CLK_DIV_PIN}] -add
+                       -divide_by 2 [get_pins ${CLK_DIV_PIN}]
```

Comment:
how come we removed the `-add` here? I thought we needed it if defining clocks on the same pin? Unless...we are not actually defining them on the same pin...

Reply:
The SDC setup checks in the CDC tool throw a warning that this command is not correct...
I'll check again why exactly.

Reply:
I believe we're not actually creating multiple clocks on the same pin, so no `add` is needed.
I.e., the division goes through a flop, which should stop any upstream clock propagation to the output if I remember correctly...

Reply:
yeah i probably at one point had multiple clocks on the same pin but removed it afterwards.  Thanks for fixing.